### PR TITLE
dicttoxml v1.7.11

### PR DESCRIPTION
**Pull request by maresb**
Bump the version, delete the patch, and fix the invalid SPDX license identifier

Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by maresb** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ about:
   license: GPL-2.0-only
   summary: Converts a Python dictionary or other native data type into a valid XML string.
   license_family: GPL
-  license_file: LICENSE.txt
+  license_file: LICENCE.txt
```
Comment (jan-janssen) [inline recipe/meta.yaml:34]: The file name has to be consistent with https://github.com/quandyfactory/dicttoxml/blob/master/LICENCE.txt

Comment (maresb) [reply recipe/meta.yaml:34]: I don't get it, but i can't fix until tomorrow, so please go ahead with any changes

Comment (maresb) [reply recipe/meta.yaml:34]: Oh spelling

**Pull request merged**